presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -616,6 +618,26 @@ public PlanWithProperties visitFilter(FilterNode node, PreferredProperties prefe
             return rebaseAndDeriveProperties(node, planChild(node, preferredProperties));
         }
 
+        @Override
+        public PlanWithProperties visitSequence(SequenceNode node, PreferredProperties preferredProperties)
+        {
+            List<PlanWithProperties> leftPlans = node.getCteProducers().stream()
+                    .map(source -> accept(source, PreferredProperties.any()))
+                    .collect(ImmutableList.toImmutableList());
```

Comment:
nit: ImmutableList.toImmutableList() -> toImmutableList(), similar for other places